movieDetail.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Movie Box</title>
+    <link rel="stylesheet" href="./assests/styleSheets/movieDetail.css" />
+  </head>
+  <body>
+    <main>
+      <div class="details-container">
+        <div class="cover-image-and-webtitle">
```

Comment:
can't we avoid this div?

Reply:
I tried but it's children are setting as row without it